src/js/actions.js

```diff
@@ -241,7 +225,6 @@ export function uploadFile(file, uiOptions, onProgress, onChange, onError) {
           name: file.name,
           errorMessage
         });
-        Raven.captureMessage(`vets_upload_error: ${req.statusText}`);
```

Comment:
This code doesn't currently have access to `formConfig` as an argument, but in addition the error is vets.gov-specific so it seems like this needs to be isolated some way. Also, we don't have file uploads implemented anyway so this code doesn't run and can't be tested.

Reply:
Probably. It's super useful functionality but we need it to be generalized so it can work with other servers.